### PR TITLE
prepare oCIS 5.0.0-rc.3

### PR DESCRIPTION
**Pull request by wkloucek**
## Description
- use 5.0.0-rc.3
- expose request concurrency settings for Web
- fix some documentation strings
- add post processing restart jobs

## Related Issue
- #473 
- https://github.com/owncloud/enterprise/issues/6358

## How Has This Been Tested?

Test setup:

```diff
diff --git a/deployments/ocis-nats/helmfile.yaml b/deployments/ocis-nats/helmfile.yaml
index ec1f0e8..0ed6902 100644
--- a/deployments/ocis-nats/helmfile.yaml
+++ b/deployments/ocis-nats/helmfile.yaml
@@ -110,6 +110,10 @@ releases:
           nodes:
             - nats.ocis-nats.svc.cluster.local:4222
 
+      - features:
+          virusscan:
+            enabled: true
+
       - services:
           idm:
             persistence:
@@ -130,6 +134,18 @@ releases:
           storageusers:
             persistence:
               enabled: true
+            maintenance:
+              cleanUpExpiredUploads:
+                enabled: true
+              purgeExpiredTrashBinItems:
+                enabled: true
+
+          postprocessing:
+            maintenance:
+              restartPostprocessingFinished:
+                enabled: true
+              restartPostprocessingVirusscan:
+                enabled: true
 
           store:
             persistence:
@@ -138,7 +154,16 @@ releases:
           thumbnails:
             persistence:
               enabled: true
+            maintenance:
+              cleanUpOldThumbnails:
+                enabled: true
 
           web:
             persistence:
               enabled: true
+            config:
+              concurrency:
+                resourceBatchActionRequests: 1
+                sseRequests: 1
+                shareCreateRequests: 1
+                shareListRequests: 1
```

- jobs, by subscribing to the main-queue in nats-box:
```
[#3] Received on "main-queue" with reply "_INBOX.IMNGj4x5eQToLmGdl8KLwB.5Wqj8jno"
{"ID":"91241888-a873-4091-aa4a-5f5a6ee0e4af","Topic":"main-queue","Timestamp":"2024-01-30T10:54:01.427013357Z","Metadata":{"eventid":"0a04049d-24bb-40b6-9ed6-ad8363600099","eventtype":"events.ResumePostprocessing","traceparent":""},"Payload":"eyJVcGxvYWRJRCI6IiIsIlN0ZXAiOiJ2aXJ1c3NjYW4iLCJUaW1lc3RhbXAiOnsic2Vjb25kcyI6MTcwNjYxMjA0MSwibmFub3MiOjQyNjk5Mzg0MH19"}


[#4] Received on "main-queue" with reply "_INBOX.sgaZVlefFO7Aq79k7WuAUU.kq1ON8Ps"
{"ID":"1f7a2f73-8eb2-49f7-adb2-2f418d5da296","Topic":"main-queue","Timestamp":"2024-01-30T10:54:01.467230399Z","Metadata":{"eventid":"a9b35b00-1081-4c2a-bbce-34d7901c6e8b","eventtype":"events.ResumePostprocessing","traceparent":""},"Payload":"eyJVcGxvYWRJRCI6IiIsIlN0ZXAiOiJmaW5pc2hlZCIsIlRpbWVzdGFtcCI6eyJzZWNvbmRzIjoxNzA2NjEyMDQxLCJuYW5vcyI6NDY3MjA5MDg4fX0="}


[#5] Received on "main-queue" with reply "_INBOX.xc9Qf6xIxJx3dR3VgQYl9G.2YCXr5Nz"
{"ID":"8cc328e8-697f-4396-b1b2-4f3f80fafb00","Topic":"main-queue","Timestamp":"2024-01-30T10:54:01.575920338Z","Metadata":{"eventid":"267fbcf9-fe82-4e2c-b73a-314839685743","eventtype":"event.PurgeTrashBin","traceparent":""},"Payload":"eyJFeGVjdXRhbnRJRCI6bnVsbCwiRXhlY3V0aW9uVGltZSI6IjIwMjQtMDEtMzBUMTA6NTQ6MDEuNTc1ODk3MTM0WiJ9"}
```
- concurrency settings by looking at /config.json, found this: https://github.com/owncloud/ocis-charts/issues/473#issuecomment-1916271493

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:
- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [x] Documentation generated (`make docs`) and committed
- [ ] Documentation ticket raised: <link>
- [ ] Documentation PR created: <link>

### CODE REVIEW

File: charts/ocis/templates/postprocessing/jobs.yaml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+              args: ["postprocessing", "restart", "-s", "finished"]
+              securityContext:
+                runAsNonRoot: true
+                runAsUser: {{ .Values.securityContext.runAsUser }}
+                runAsGroup: {{ .Values.securityContext.runAsGroup }}
+                readOnlyRootFilesystem: true
+              env:
+                {{- include "ocis.serviceRegistry" . | nindent 16 }}
+                {{- include "ocis.events" . | nindent 16 }}
+                {{- include "ocis.persistentStore" . | nindent 16 }}
```
Comment (kobergj) [inline charts/ocis/templates/postprocessing/jobs.yaml:42]: The command only needs the `ocis.events` configuration. It does not need access to the `serviceRegistry` and the `persistentStore`

File: charts/ocis/templates/postprocessing/jobs.yaml [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+              args: ["postprocessing", "restart", "-s", "virusscan"]
+              securityContext:
+                runAsNonRoot: true
+                runAsUser: {{ .Values.securityContext.runAsUser }}
+                runAsGroup: {{ .Values.securityContext.runAsGroup }}
+                readOnlyRootFilesystem: true
+              env:
+                {{- include "ocis.serviceRegistry" . | nindent 16 }}
+                {{- include "ocis.events" . | nindent 16 }}
+                {{- include "ocis.persistentStore" . | nindent 16 }}
```
Comment (kobergj) [inline charts/ocis/templates/postprocessing/jobs.yaml:98]: The command only needs the `ocis.events` configuration. It does not need access to the `serviceRegistry` and the `persistentStore`

**Review by kobergj — APPROVED** [review]
Looks good. Some configuration could be removed. But not critical

File: charts/ocis/templates/postprocessing/jobs.yaml (hunk 1 above)
Comment (wkloucek) [reply charts/ocis/templates/postprocessing/jobs.yaml]: great find, fixed it :+1:

File: charts/ocis/templates/postprocessing/jobs.yaml (hunk 2 above)
Comment (wkloucek) [reply charts/ocis/templates/postprocessing/jobs.yaml]: great find, fixed it :+1:

**Pull request merged**